src/go/bootstrap/ads/bootstrap_test.go

```diff
@@ -115,7 +116,7 @@ func TestCreateBootstrapConfig(t *testing.T) {
 			desc: "bootstrap with options",
 			args: map[string]string{
 				"disable_tracing": "true",
-				"enable_admin":    "true",
+				"admin_port":      "8001",
```

Comment:
Nit: Clean up this test to set Options instead of flags. Then you don't need to reset the flags in the following tests.

Reply:
I think ads is using flag to generate the config, different from static one. Should be fine.